checks/deprecated-define.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class Deprecated_Define implements themecheck {
+	protected $error = array();
+
+	function check( $php_files, $css_files, $other_files ) {
+		$grep = '';
+
+		$ret = true;
+
+		$checks = array(
+			array( "NO_HEADER_TEXT" => "add_theme_support( 'custom-header' )", '3.4' ),
+			array( "HEADER_IMAGE" => "add_theme_support( 'custom-header' )", '3.4' ),
+			array( "HEADER_IMAGE_WIDTH" => "add_theme_support( 'custom-header' )", '3.4' ),
+			array( "HEADER_IMAGE_HEIGHT" => "add_theme_support( 'custom-header' )", '3.4' ),
+			array( "HEADER_TEXTCOLOR" => "add_theme_support( 'custom-header' )", '3.4' ),
+			array( "BACKGROUND_COLOR" => "add_theme_support( 'custom-background' )", '3.4' ),
+			array( "BACKGROUND_IMAGE" => "add_theme_support( 'custom-background' )", '3.4' ),
+		);
+		foreach ( $php_files as $php_key => $phpfile ) {
+			foreach ( $checks as $alt => $check ) {
+				checkcount();
+				$key = key( $check );
+				$alt = $check[ $key ];
+				if ( preg_match( '/' . $key . '/', $phpfile, $matches ) ) {
```

Comment:
You could try `preg_match( '/(?:'|")' . $key . '(?:'|")/', $phpfile, $matches )`